Make.inc

```diff
@@ -1394,13 +1394,13 @@ define symlink_target # (from, to-dir, to-name)
 CLEAN_TARGETS += clean-$$(abspath $(2)/$(3))
 clean-$$(abspath $(2)/$(3)):
 ifeq ($(BUILD_OS), WINNT)
-	-cmd //C rmdir $$(call mingw_to_dos,$(2)/$(3),cd $(2) &&)
```

Comment:
We don't need `mingw_to_dos` anymore. Do we need to remove this function?

Reply:
Yes, let's remove the function if we're no longer calling it.

Reply:
It appears to me that we're still using this in `deps/tools/common.mk`, so I'm not going to remove the Makefile function.